### PR TITLE
Issue463 abstract test case tests

### PR DESCRIPTION
**Pull request by EttoreZ**
This pull request is to address issue #463.

### CODE REVIEW

File: testing/test_bestest_air.py [hunk 1]
```diff
@@ -60,6 +60,8 @@ def setUp(self):
         self.url = 'http://127.0.0.1:5000'
         self.step_ref = 3600
         self.test_time_period = 'peak_heat_day'
+        self.input = {'fcu_oveTSup_activate': 0, 'fcu_oveTSup_u': 290}
```
Comment (dhblum) [inline testing/test_bestest_air.py:63]: @EttoreZ I suggest calling "input_false_overwrite" or add a comment that the value of activate is meant to be 0 for the test_advance_false_overwrite API test.

**Review by dhblum — CHANGES_REQUESTED** [review]
@EttoreZ Thanks for this PR.  Please address the one inline comment and then I think this will be ready to go.

**Comment by EttoreZ** [comment]
Thank you @dhblum for the careful review. I added the comment `#<u_variable>_activate is meant to be 0 for the test_advance_false_overwrite API test` above the self.input definition and updated the pull request.

**Review by dhblum — APPROVED** [review]
Thanks @EttoreZ for the udpates.

**Pull request merged**